libs/sdk-ui-dashboard/src/presentation/dashboard/types.ts

```diff
@@ -578,6 +578,29 @@ export interface IDashboardEventing {
     onStateChange?: (state: DashboardState, dispatch: DashboardDispatch) => void;
 }
 
+/**
+ * @internal
+ */
+export enum DashboardPartIds {
+    MainContent = "mainContent",
+    FiltersBar = "filtersBar",
+    Sidebar = "sidebar",
+}
+
+/**
+ * @internal
+ */
+export interface INavigationConfigItem {
+    tabIndex?: number;
+    navigationId?: string;
```

Comment:
Maybe `elementId` would convey that this is ID that will be used for the element?

Reply:
Possibly, I would add a TSDoc comment that will explain what is this ID used for.